egs/wsj/s5/steps/libs/nnet3/train/common.py

```diff
@@ -530,6 +731,30 @@ def __init__(self):
                                  Note: we implemented it in such a way that it
                                  doesn't increase the effective learning
                                  rate.""")
+        self.parser.add_argument("--trainer.dropout-schedule", type=str,
+                                 dest='dropout_schedule', default='',
```

Comment:
default value here better be None, when run `steps/nnet3/chain/train.py`, your dropout default type is None:
```
    if args.dropout_schedule is not None:
        dropout_schedule = common_train_lib.parse_dropout_option(
            num_archives_to_process, args.dropout_schedule)
```
 not null string, this will conflict.